.gitignore

```diff
@@ -4,3 +4,6 @@ out
 coverage
 test/**/dist
 test/**/actual.js
+package-lock.json
+yarn.lock
+pnpm-lock.yaml
```

Comment:
@cb1kenobi  I think we should keep one lockfile here so that we have deterministic behavior.

Reply:
Fixed!